df.interaction.xml

```diff
@@ -307,9 +307,9 @@
 
     <struct-type type-name='interaction_instance' instance-vector='$global.world.interaction_instances.all' key-field='id'>
         <int32_t name='id'/>
+        <int32_t name='interation_id'/>
```

Comment:
From the forum post, I think this should also have a ref-target attribute (but I can add that later).